packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -210,19 +210,33 @@ export default class Meeting extends StatelessSparkPlugin {
     this.mediaId = locus.mediaId;
   }
 
-  setRemoteStreams(pc) {
+  setRemoteStream(pc) {
     // Receiving remote media streams
     pc.ontrack = (event) => {
-      [this.mediaStream, this.localMediaStream] = event.streams;
+      [this.remoteStream] = event.streams;
 
+      // TODO: It's possible for media to not be present
+      // so we might need to either
+      // A) wait until we have media flowing
+      // B) trigger a second event when video is flowing
       this.trigger('media:ready', {
         type: 'remote',
-        stream: this.mediaStream
+        stream: this.remoteStream
       });
     };
   }
 
-  setScreenStreams(pc) {
+  unsetRemoteStream() {
+    return Media.stopStream(this.remoteStream).then(() => {
+      this.remoteStream = null;
+
+      this.trigger('media:stopped', {
```

Comment:
i can fix this, but we should change to the constant EVENT_TRIGGERS.MEDIA_STOPPED

Reply:
Yeah I saw you changed the others and was planning to update after your PR was merged and I rebased on it